desktop/wallet/main.js

```diff
@@ -229,30 +205,88 @@ function downloadUpdate($event, assetUrl) {
 function onReady() {
   createWindow();
 
-  if (win) {
+  if (win && win.webContents) {
     win.webContents.on('did-finish-load', () => {
       updateService.start((newVersion) => {
         if (newVersion) {
           handleLatestUpdate(newVersion);
         }
       });
+
+      // Deeplinks for initial startup
+      process.argv.forEach((arg) => {
+        if (/burst:\/\//.test(arg)) {
```

Comment:
```suggestion
        if (/^burst:\/\//.test(arg)) {
```

otherwise would match `fooburst://` and many others :grin: